deploy-stg.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+timestamp=$(date +%s)
+
+# remember initial branch
+dev_branch=$(git rev-parse --abbrev-ref HEAD)
+
+# replace possible "/" with "-"
```

Comment:
Could you include a comment why it is done? Gitlab does not like `/` in branch names?